<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubscriptionWindow</class>
 <widget class="QWidget" name="SubscriptionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="4,8">
   <item row="0" column="0">
    <widget class="QLabel" name="labelSubscription">
     <property name="text">
      <string>Subscription:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelSubscriptionDisplay">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelStatusDisplay">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelQos">
     <property name="text">
      <string>QoS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelQosDisplay">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonUnsubscribe">
     <property name="text">
      <string>Unsubscribe</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
